lib/known_bugs.pm

```diff
@@ -28,12 +28,21 @@ our @EXPORT = qw(
   upload_journal
 );
 
+=head1 SYNOPSIS
```

Comment:
This line is more useful if it contains the name of the module, when the doc is generated apart from the code.

```suggestion
=head1 utils/known_bugs
```

Reply:
In utils.pm we have this style somehow established, if your intention is to clarify what module we are on, this could be done via the page title?
Where else would you place the synopsis? On an own head2?